docs/package.json

```
@@ -10,6 +10,7 @@
     "test": "$_ run build"
   },
   "dependencies": {
+    "@aws-amplify/ui-react": "^2.1.5",
```

@wlee221 I'm curious if Changesets will automatically update this dependency 🤔 

Alternatively, this could be

```suggestion
    "@aws-amplify/ui-react": "latest",
```

or

```suggestion
    "@aws-amplify/ui-react": "*",
```

Whichever works for both local development (by not actually installing the NPM dependency) and Amplify Hosting.

---

Changesets will! It keeps track of cross dependencies. We can do a sanity check by 

1) add changeset to bump versions:

```
cp .github/changeset-presets/bump-versions.md .changeset
```

2) version packages

```
yarn changeset version
```

3) Check that docs' `@aws-amplify/ui-react` version is bumped

---

I'll do that while we wait for preview deployment.

---

Mocked version bump here: https://github.com/aws-amplify/amplify-ui/compare/add-missing-react-dependency...add-missing-react-dependency-test-version-bump. Looks good!

---

Also confirmed that `node_modules/@aws-amplify/ui-react` symlinks to local package.

---

Thanks @wlee221 !  Let's keep this as-is then 👍